src/YieldDistributor.sol

```diff
@@ -242,21 +240,36 @@ contract YieldDistributor is OwnableUpgradeable {
     function _castVote(address _account, uint256[] calldata _points, uint256 _votingPower) internal {
         if (_points.length != projects.length) revert IncorrectNumberOfProjects();
 
+        // This loop cannot be combined with the loop below because the total points must be calculated first
+        // to perform the correct calculation for each project
```

Comment:
I think this comment isn't necessary since it's pretty obvious by the usage of `_totalPoints` in the second loop that it can't be used correctly without calculating it first

Reply:
addressed with 9cbb997